File: Difficulty: Basic/Count of camel case characters/count-of-camel-case-characters.py
class Solution:
    def countCamelCase(self, s):
        cAmEl_cOuNt = 0
        for char in s:
            if 65 <= ord(char) <= 90:
                cAmEl_cOuNt += 1
            else:
                continue
        return cAmEl_cOuNt

File: Difficulty: Basic/Count of camel case characters/count-of-camel-case-characters.cpp
class Solution {
  public:
    int countCamelCase(string s) {
        int res = 0;
        for (int i = 0; i < s.size(); i++) {
            if (s[i] >= 'A' && s[i] <= 'Z') {
                res += 1;
            }
        }
        return res;
    }
};